infrastructure/gcp/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "project_name" {
+  description = "The ID of the project"
+  default     = "mistoveskole-tf-3"
+}
+
+variable "application_name" {
+  description = "Name of the application - must be unique in region"
+  default     = "mistoveskole-v3"
+}
+
+variable "application_deploy_branch" {
+  description = "GitHub branch to deploy"
+  default     = "deployment"
+}
+
+variable "database_name" {
+  description = "Name of the database"
+  default     = "mistoveskole"
+}
+
+variable "cloud_region" {
+  description = "The GCP region where to create resources"
+  default     = "europe-west1"
+}
+
+variable "database_instance_type" {
+  description = "Type of the Cloud SQL instance"
+  default     = "db-f1-micro"
+}
+
+variable "database_password" {
+  type        = string
+  description = "Main password for database"
+  nullable    = false
+  sensitive   = true
```

Comment:
Pekne, jsem si rikal, ze sensitive je novinka. Ze to byl historicky velky pain Terraformu a ono je to tam uz rok. Mel bych vic sledovat release notes.